components/Header.js

```diff
@@ -158,11 +152,9 @@ export default function Header() {
               </div>
               <div className={`group h-full hover:bg-blue-850 ${router.pathname == '/alumni' && "bg-blue-850"}`}>
                 <div className="md:flex md:flex-col md:justify-center md:h-full">
-                  <Link href="">
-                    <a className="py-4 px-2 text-white text-sm">
-                      ALUMNI
-                    </a>
-                  </Link>
+                  <a className="py-4 px-2 text-white text-sm cursor-pointer">
+                    ALUMNI
```

Comment:
What is the reason for converting a link to an anchor tag?

Reply:
Before this change, when we click on this, it changes the scrollable component at top.

Before click, page is shown as:
![image](https://user-images.githubusercontent.com/44665645/135036765-a8a8b3f7-a0fc-4d2c-b496-f67ac61d6f99.png)

After click, it is forwarding to top:
![image](https://user-images.githubusercontent.com/44665645/135036863-b5088b34-31d9-41d6-8798-2a61230b2807.png)

Due to this reason, I think to remove href and make it text but Link tag needs an href so I removed that.

If this is not an issue, then I will add Link tag again.

Reply:
Got it!
Thank you for this. I was about to open a separate issue for this.